.gitignore

```diff
@@ -75,6 +75,7 @@ docs/manual/figures/signature-types-with-canonicalname.dia
 docs/manual/*.png
 docs/manual/*.svg
 docs/examples/**/*.class
+docs/manual/manual.html-e
```

Comment:
OK, thanks for the explanation.  I wish I knew what created the file, but given your report, it's best to retain the line in the `.gitignore` file.